lib/layouts/SubHeader.js

```diff
@@ -0,0 +1,230 @@
+import React, { useState, useRef, useEffect } from "react";
+import * as R from "ramda";
+import classnames from "classnames";
+import { Checkbox, Tab, Button, Dropdown, Input, Pagination } from "lib";
+
+export const SORT_ORDER = [
+  { label: "Ascending", value: "asc" },
+  { label: "Descending", value: "desc" }
+];
+
+const SubHeader = ({
+  searchProps,
+  columnFilterProps,
+  tabsProps,
+  deleteButtonProps,
+  paginationProps,
+  sortProps,
+  changeStateProps,
+  className,
+  toggleFilter,
+  actionBlock
+}) => {
+  const searchRef = useRef();
+  const [searchVisibility, setSearchVisibility] = useState(
+    searchProps ? !searchProps.clear : false
+  );
+
+  const searchParams = new URLSearchParams(location.search);
+
+  const [sortBy, setSortBy] = useState({
+    column:
+      searchParams.get("sort") ||
+      (sortProps && sortProps.options[0] && sortProps.options[0].value),
+    direction: searchParams.get("order") || SORT_ORDER[1].value
+  });
+
+  useEffect(() => {
+    sortProps && sortProps.onClick(sortBy);
+  }, [sortBy]);
+
+  const handleClick = (column, direction) => {
+    setSortBy({
+      ...sortBy,
+      column,
+      direction
+    });
+  };
+
+  const handleSearchVisibilty = status => {
+    setSearchVisibility(status);
+    if (status) {
+      searchRef.current.focus();
+    } else {
+      searchProps && searchProps.clear();
+    }
+  };
+
+  const currentSort =
+    sortProps &&
+    sortProps.options.find(item => item.value === sortBy.column).label;
+
+  const showColumnFilters = R.all(R.either(R.isNil, R.isEmpty), [
+    columnFilterProps,
+    columnFilterProps?.columnFilters,
+    columnFilterProps?.handleColumnFiltering
+  ]);
+
+  const ColumnFiltersDropdown = () => {
+    const disabledColumn = "name";
```

Comment:
This should be removed.

Reply:
@goutham-subramanyam 

Can you clarify this a bit more? Did you mean to remove the check for `disabledColumn` all together or no need to sync this file?

Reply:
The `disabledColumn` check should be removed.